migrations/sql/V2020.11.05.13.09__create_now_delay.sql

```diff
@@ -0,0 +1,31 @@
+
+    CREATE TABLE IF NOT EXISTS now_application_delay_type ( 
+        delay_type_code varchar PRIMARY KEY,
+        description varchar,
+        display_order integer,
+        active_ind boolean NOT NULL default 'True',
```

Comment:
`default true`?

Reply:
Postgres accepts a wide variety of things that i can interpret as 'truthy'. I will double check this one though. 
https://www.postgresql.org/docs/9.6/datatype-boolean.html